templates/README.md

```diff
@@ -0,0 +1,5 @@
+This directory contains a few reusable files for coq-community (or external)
+Coq projects.
+
+Files ending with `.template` have values to fill in to use
+(and the `.template` extension should be removed).
```

Comment:
Perhaps we could actually use a template language like [mustache](https://mustache.github.io/)? Since we don't need any logic, it shouldn't clutter up the template any. In fact, it might make it easier to tell what needs to be replaced, and eases any potential automation.